j2lint/linter/error.py

```diff
@@ -1,16 +1,18 @@
 """error.py - Error classes to format the lint errors.
 """
+import json
 from j2lint.settings import settings
 from j2lint.logger import logger
-import json
+
+# pylint: disable=too-few-public-methods,too-many-arguments
```

Comment:
Better to disable too-many-arguments at the method level rather than at the file level